xgboost-job/xgboost-operator/base/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- cluster-role.yaml
+- cluster-role-binding.yaml
+- crd.yaml
+- deployment.yaml
+- service-account.yaml
+- service.yaml
+namespace: kubeflow
+nameprefix: xgboost-operator-
+configMapGenerator:
+- name: parameters
```

Comment:
I think you want to use a unique enough name so that it won't conflict with configmaps from other applications.
https://github.com/kubeflow/manifests/blob/5de67258ca92a59f29d025d69a566310fce208f5/jupyter/jupyter-web-app/base_v3/kustomization.yaml#L46

We've been using the pattern

${appname}-config